lib/src/http_client_response.dart

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2021, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+part of '../src/http_client_request.dart';
+
+/// The body of a [HttpClientResponse] object is a [Stream] of data from the server.
+/// Listen to the body to handle the data and be notified when the entire body is received.
```

Comment:
Doc comments start with a single line explanation.

And then one or more paragraphs.

(Add a newline.)